src/test/java/com/google/devtools/build/lib/BUILD

```diff
@@ -1167,7 +1167,7 @@ java_test(
 
 java_test(
     name = "remote-tests",
-    srcs = glob(["remote/*.java"]),
+    srcs = glob(["remote/**/*.java"]),
```

Comment:
Prefer splitting out the http part into a separate library. Best practice is one BUILD file per package.

Reply:
done